src/base_DAO.mligo

```diff
@@ -53,14 +51,7 @@ let allowing_xtz (param, store, config : allow_xtz_params * storage * config) =
  *)
 let requiring_no_migration (param, store, config : migratable_parameter * storage * config)
```

Comment:
note that with the other suggestion this can be merged into `base_DAO_contract` (as there are no longer entrypoints requiring no migration). Indeed its comment above is already no longer accurate.